mlflow/store/artifact/cloud_artifact_repo.py

```diff
@@ -29,6 +29,15 @@
 _ARTIFACT_UPLOAD_BATCH_SIZE = (
     50  # Max number of artifacts for which to fetch write credentials at once.
 )
+_AWS_MIN_CHUNK_SIZE = 5 * 1024**2  # 5 MB is the minimum chunk size for S3 multipart uploads
+
+
+def _validate_chunk_size_aws() -> None:
+    chunk_size = MLFLOW_MULTIPART_UPLOAD_CHUNK_SIZE.get()
+    if chunk_size <= _AWS_MIN_CHUNK_SIZE:
```

Comment:
can we also validate the chunk size doesn't exceed the upper limit? 

https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html